content/vue/step02.md

```diff
@@ -0,0 +1,124 @@
+
+### Replace the starter code
+
+To get started, let's replace the code of the default starter app. Then we'll talk about what it does.
+
+First, replace the content of the **`client/main.html`** file:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.2"}}
+
+Second, replace the contents of the **`client/main.js`** module:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.3"}}
+
+Now we need to create a new directory called `imports`, with a directory called `ui` inside of it. There will soon be other directories besides `ui` within `imports`, but we'll start with `ui`.
+
+> Note: in previous versions of Meteor, the `imports` directory was special because files outside the `imports` directory were loaded automatically when the application started, whereas files inside the `imports` directory were only loaded when imported using an `import` declaration or a `require` statement. As of Meteor 1.7, the entry point for both client and server JavaScript is determined by the `meteor.mainModule` section in `package.json`. In other words, as far as JavaScript code is concerned, the entire application now behaves as if it was inside an `imports` directory, so you don't need to worry as much about the `imports` directory now.
+
+To continue converting your app to use Vue, copy the following code into **`imports/ui/App.vue`**:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.4"}}
+
+Now copy the following code into **`imports/ui/Task.vue`**:
+
+{{> DiffBox tutorialName="simple-todos-vue" step="2.5"}}
+
+We just added three things to our app:
+
+1. An `App` Vue component in `imports/ui/App.vue`
+2. A `Task` Vue component in `imports/ui/Task.vue`
+3. Some initialization code (in our `client/main.js` client JavaScript entrypoint), in a `Meteor.startup` block, which knows how to call code when the page is loaded and ready. This code creates the root Vue instance and passes in the options object to it's contructor. This root Vue instance will be mounted using the `#app` html element.
+
+You can read more about how imports work and how to structure your code in the [Application Structure article](http://guide.meteor.com/structure.html) of the Meteor Guide.
+
+Later in the tutorial, we will refer to these components when adding or changing code.
+
+### Check the result
+
+In our browser, the app should **roughly** look like the following (though much less pretty):
+
+> #### Todo List
+>
+> - This is task 1
+> - This is task 2
+> - This is task 3
+
+If your app doesn't look like this, use the GitHub link at the top right corner of each code snippet to see the entire file, and make sure your code matches the example.
+
+### HTML files define static content
+
+Meteor parses all of the HTML files in your app folder and identifies three top-level tags: **`<head>`**, **`<body>`**, and **`<template>`**.
```

Comment:
Isn't this only true as far as the import rules are concerned? Just mentioning it because you're here talking about **all of the HTML files**. I couldn't imagine this overriding the import rules defined earlier.

E.g. if I use the section `meteor.mainModule` in my `package.json` file, this only applies to the HTML files I import using an import statement, doesn't it?